web-wallet/src/lib/contracts/areValidSettings.js

```diff
@@ -0,0 +1,27 @@
+import { gasStore } from "$lib/stores";
+import { isType } from "lamb";
+import { get } from "svelte/store";
+
+const gasLimits = get(gasStore);
+
+/**
+ *
+ * @param {Number} price
+ * @param {Number} limit
+ * @returns {Boolean}
+ */
+const areValidSettings = (price, limit) => {
```

Comment:
As we moved the function out of a "gas context", maybe it's better to add "gas" back into the function name?
e.g. `areValidGasSettings` / `areGasSettingsValid`?